docs/examples/mention/loading.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <el-mention
+    v-model="value"
+    :options="options"
+    :loading="loading"
+    style="width: 320px"
+    placeholder="Please input"
+    @search="handleSearch"
+  />
+</template>
+
+<script setup lang="ts">
+import { ref } from 'vue'
+import type { MentionOption } from 'element-plus'
+
+const value = ref('')
+const loading = ref(false)
+const options = ref<MentionOption[]>([])
+
+let timer: ReturnType<typeof setTimeout>
```

Comment:
The timer needs to be removed in onBeforeUnmount

Reply:
done